packages/workflow/src/Expression.ts

```diff
@@ -278,10 +278,6 @@ export class Expression {
 					throw error;
 				}
 			}
-
-			if (error instanceof Error && error.name === 'SyntaxError') {
```

Comment:
are there no other kinds of errors possible here that we need to rethrow ?

Reply:
We are ignoring all errors except expression errors that have `failExecution`. If we throw anything that we were not throwing before (e.g. this syntax error) it will be a breaking change and it was decided [not to make any breaking changes](https://n8nio.slack.com/archives/C035KBDA917/p1671634999527279 ) at this time.